src/main/java/org/cryptomator/common/settings/VaultSettings.java

```diff
@@ -138,11 +138,14 @@ public StringProperty customMountPath() {
 	}
 
 	public Optional<String> getCustomMountPath() {
-		if (useCustomMountPath.get()) {
-			return Optional.ofNullable(Strings.emptyToNull(customMountPath.get()));
-		} else {
+		if (!useCustomMountPath.get()) {
 			return Optional.empty();
 		}
+		String mountPath = Strings.emptyToNull(customMountPath.get());
+		if (mountPath == null) {
+			throw new IllegalArgumentException("Error K5G8:EJ0V:EJ0V - No custom mountpath set!");
```

Comment:
Don't add an error code to a exception message. The actual error code depends on the call stack and we don't want to end up with an error containing a different error code.

Reply:
Turns out this exception prohibits change of mount settings, as this method is called during initialization of the vault options dialog.

As long as the GUI doesn't prevent this state, it must not throw.